build-logic/src/main/kotlin/polaris-runtime.gradle.kts

```diff
@@ -47,6 +54,23 @@ testing {
   }
 }
 
+dependencies {
+  // All Quarkus projects should use JBoss LogManager with SLF4J, instead of Logback
+  implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
+}
```

Comment:
You may want something like this to ban logback from Quarkus.
```
configurations.all { exclude(group = "ch.qos.logback", module = "logback-classic") }
```

BTW: it would also need to be removed from the LICENSE/NOTICE